### PR TITLE
Allow to run aiidalab image on local computer.

### LINKED ISSUE #37: should containers work outside JupyterHub
@espenfl notes that 
> with the aiida one I get `JUPYTERHUB_API_TOKEN env is required to run jupyterhub-singleuser`. 

Perhaps it would make sense if one could run the aiidalab docker container also outside the JupyterHub?

### PR DESCRIPTION
**Pull request by yakutovicha**
fixes #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,48 +4,61 @@ This repo contains the Docker file used in the [AiiDA lab](https://www.materials
...
+```
+Usage:	docker [OPTIONS] COMMAND
+
+A self-sufficient runtime for containers
+
+Options:
+...
+```
+
+If this is the case, everything proceed with the next steps. Otherwise, please visit the [Docker installation page](http://www.docker.com/install) and install Docker on your machine.
```
Comment (ltalirz) [inline README.md]: I'd simplify this and simply say to install docker (with link)

File: README.md [hunk 2]
```diff
@@ -4,48 +4,61 @@ This repo contains the Docker file used in the [AiiDA lab](https://www.materials
...
+
+To pull image from DockerHub and start the container do the following:
```
Comment (ltalirz) [inline README.md]:
```suggestion
Then, pull the `aiidalab-docker-stack` image from DockerHub and start a container:
```

File: README.md [hunk 3]
```diff
@@ -4,48 +4,61 @@ This repo contains the Docker file used in the [AiiDA lab](https://www.materials
...
+```
+docker pull aiidalab/aiidalab-docker-stack:latest
+mkdir ${HOME}/aiidalab # Create a new folder where your data vill be stored
+docker run -d -p 8888:8888 -v ${HOME}/aiidalab:/home/aiida aiidalab/aiidalab-docker-stack:latest
+# Note the Docker ID that was displayed after the last command and put it into the following run
```
Comment (ltalirz) [inline README.md]: Wouldn't it be easier to encapsulate the steps from `docker run` in a script?

File: README.md (hunk 1 above)
Comment (yakutovicha) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (yakutovicha) [reply README.md]: done.

File: run_aiidalab.sh [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+docker pull ${IMAGE}
+
+echo "Launching the container..."
+CONTAINERID=`docker run -d -p 8888:${PORT} -e JUPYTER_TOKEN=${TOKEN} -v ${FOLDER}:/home/aiida ${IMAGE}`
+
+echo "Waiting for container to start..."
+docker exec --tty ${CONTAINERID} wait-for-services
+
+echo "Container started successfully."
+echo "Put this link to browser to run AiiDA lab:"
```
Comment (ltalirz) [inline run_aiidalab.sh]:
```suggestion
echo "Open this link in the browser to enter AiiDA lab:"
```

**Pull request merged**